tools/export/__init__.py

```diff
@@ -93,12 +96,13 @@ def export(project_path, project_name, ide, target, destination='/tmp/',
                     use_progen = True
             except AttributeError:
                 pass
+
+            if target not in Exporter.TARGETS or ide.upper() not in TARGET_MAP[target].supported_toolchains:
+                supported = False
```

Comment:
@0xc0170 
What do you think about this? Not sure why progen checks were different. Maybe we should think about getting rid of the hard coded targets in gcc, etc?